content-repo/create-docs.sh

```diff
@@ -144,9 +144,21 @@ if [ -z "${NETLIFY}" ]; then
     pipenv install
     echo "Generating docs..."
     pipenv run ./gendocs.py -t "${TARGET_DIR}" -d "${CONTENT_GIT_DIR}"
+    echo "Generating Demisto class docs..."
+    echo "---" >> ../docs/integrations/demisto-class.md
+    echo "id: demisto-class" >> ../docs/integrations/demisto-class.md
+    echo "title: Demisto Class" >> ../docs/integrations/demisto-class.md
+    echo "---" >> ../docs/integrations/demisto-class.md
+    mv demistomock.py demisto.py
+    pipenv run pydoc-markdown -p demisto >> ../docs/integrations/demisto-class.md
```

Comment:
@Itay4 to avoid all this duplicate code and have a better framework let's move all of this into 1 python script `gen-pydocs.py`. That way we can also add unit testing as we evolve and so on.